garden_ai/_model.py

```diff
@@ -99,13 +99,32 @@ def clear_mlflow_staging_directory():
     def _lazy_load_model(self):
         """download and deserialize the underlying model, if necessary."""
         import mlflow  # type: ignore
+        import yaml
 
         if self._model is None:
             download_url = self.get_download_url(self.full_name)
             local_model_path = self.download_and_stage(download_url, self.full_name)
-            self._model = mlflow.pyfunc.load_model(
-                local_model_path, suppress_warnings=True
-            )
+            mlflow_yaml_path = local_model_path + "/MLmodel"
+            with open(mlflow_yaml_path, "r") as stream:
+                mlflow_metadata = yaml.safe_load(stream)
+            mlflow_load_strategy = mlflow_metadata["metadata"]["garden_load_strategy"]
```

Comment:
Will this give a KeyError for old models? (Not a deal breaker, we can do another DB wipe. We just have to be sure to do it.)